espei/run_espei.py

```diff
@@ -131,16 +131,19 @@ def main():
         save_interval = mcmc_settings.get('mcmc_save_interval')
         # scheduler setup
         if mcmc_settings['scheduler'] == 'MPIPool':
+            # check that cores is not an input setting
+            if mcmc_settings['cores'] != None:
+                logging.info("MPI does not take the cores input setting.")
```

Comment:
I think warning is more fitting. See http://espei.org/en/latest/contributing.html#logging